macro/profiler/src/lib.rs

```diff
@@ -0,0 +1,2 @@
+#[cfg(any(target_os = "windows", target_os = "macos", target_os = "linux"))]
```

Comment:
Since we only provide instructions for `macos` and `linux`, maybe limit to that for now.

Reply:
removed in https://github.com/nexus-xyz/nexus-zkvm/commit/b1b02fcafcfa15b5d3ee6c80c55a704117db836c